src/lib/pages/frontsite/Documents/DocumentDetails/DocumentCirculation/DocumentEItems.js

```diff
@@ -14,10 +15,42 @@ export class DocumentEItems extends Component {
   };
 
   onClickEItemRequestLink = () => {
+    const {
+      documentDetails: { metadata },
+    } = this.props;
     const medium = 'DIGITAL';
+
+    const formData = { medium };
+
+    const authors = _get(metadata, 'authors[0].full_name');
+    const title = _get(metadata, 'title');
```

Comment:
Many of these fields are mandatory in the Document record, so you can avoid to use `_get` for those. Have a look [here](https://github.com/inveniosoftware/invenio-app-ils/blob/master/invenio_app_ils/documents/schemas/documents/document-v1.0.0.json#L5) to find out which ones are mandatory.